package.json

```diff
@@ -50,6 +52,7 @@
 		"eslint-plugin-tsdoc": "^0.2.17",
 		"husky": "^8.0.3",
 		"lint-staged": "^13.2.0",
+		"npm-run-all": "^4.1.5",

```

Comment:
what is this for?

Reply:
this is required for the docker config.
without this we can't start the bot :/

Reply:
I thought we didn't need it but we do